code/game/machinery/suit_storage_unit.dm

```diff
@@ -54,10 +57,8 @@
 	var/message_cooldown
 	/// How long it takes to break out of the SSU.
 	var/breakout_time = 30 SECONDS
-	/// Power contributed by this machine to charge the mod suits cell without any capacitors
-	var/base_charge_rate = 200 KILO WATTS
```

Comment:
Should leave this as a var so subtypes can edit it and admins can VV it

Reply:
But we don't have enough subtypes for suit storage. But yeah I'll leave it for admins then